packages/js/src/plugins/auctionHouseModule/AuctionHouseClient.ts

```diff
@@ -236,6 +249,11 @@ export class AuctionHouseClient {
       .getTask(getBuyerBalanceOperation(options));
   }
 
+  /** {@inheritDoc saleOperation} */
+  sell(options: SellInput): Task<SellOutput> {
+    return this.metaplex.operations().getTask(sellOperation(options));
+  }
+
```

Comment:
`sell` as a method name is great because it's in the context of the auction house client.

However, `SellOperation`, `SellInput`, etc. needs to be more specific because they are exported globally (without any context).

The same goes for the `buy` operation.

May I suggest:
- `AuctionHouseDirectSellOperation`
- `AuctionHouseDirectBuyOperation`

Please update everything accordingly: transaction builder, inputs, outputs, file names, etc. Just keep the methods `sell` and `buy` as-is.

Reply:
Maybe just DirectSellOperation would be enough? As for me, it is obvious from here that it is AuctionHouse operations

Reply:
Okay let's go with `DirectSell` and `DirectBuy` 👌